src/ol/layers/MapboxLayer.js

```diff
@@ -173,6 +191,13 @@ export default class MapboxLayer extends Layer {
     );
   }
 
+  /**
+   * Returns a style URL with apiKey & apiKeyName infos.
+   */
+  createStyleUrl() {
```

Comment:
useless function , use directly getMapboxStyle

Reply:
We actually need to be able to call this function in target app so we keep this fct.